src/com/owncloud/android/files/FileMenuFilter.java

```diff
@@ -134,30 +137,34 @@ private void hideAll(Menu menu) {
      * @param toHide            List to save the options that must be shown in the menu.
      */
     private void filter(List<Integer> toShow, List <Integer> toHide) {
+
         boolean synchronizing = anyFileSynchronizing();
 
+        boolean videoPreviewing = anyFileVideoPreviewing();
```

Comment:
Not really, this variable is used to check if the file is being previewed, with streaming or not. In other part of this file I use the code below to check if the file is being streamed:

`!anyFileDown() && anyFileVideoPreviewing()`